packages/code-snippet/src/CodeSnippetService.ts

```diff
@@ -59,4 +63,26 @@ export class CodeSnippetService {
 
     return codeSnippetsByLanguage;
   }
+
+  deleteCodeSnippet(
+    codeSnippet: ICodeSnippet,
+    updateSnippets: () => void
+  ): void {
+    showDialog({
+      title: `Delete "${codeSnippet.displayName}" snippet?`,
```

Comment:
I think placing the snippet name on the end might read better since the display name could read like a sentence in some cases.  Not sure if we can remove the double-quotes (as suggested below) or not.  That's more aesthetics so we'll just need to feel that out.
```suggestion
      title: `Delete snippet: ${codeSnippet.displayName}?`,
```